From: Chris Hofstaedtler <zeha@debian.org>
Date: Thu, 14 Apr 2022 11:47:20 +0000
Subject: Install lsfd into /usr/bin

---
 lsfd-cmd/Makemodule.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lsfd-cmd/Makemodule.am b/lsfd-cmd/Makemodule.am
index d28f869..a64a9db 100644
--- a/lsfd-cmd/Makemodule.am
+++ b/lsfd-cmd/Makemodule.am
@@ -2,7 +2,7 @@ if BUILD_LSFD
 
 lsfd-cmd/file.c: errnos.h
 
-bin_PROGRAMS += lsfd
+usrbin_exec_PROGRAMS += lsfd
 MANPAGES += lsfd-cmd/lsfd.1
 dist_noinst_DATA += lsfd-cmd/lsfd.1.adoc
 lsfd_SOURCES = \
